logic.py

```diff
@@ -7,15 +7,65 @@
 
 app = Flask(__name__)
 
-app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
-app.config['MYSQL_USER'] = 'Osama'
-app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
+# app.config['MYSQL_USER'] = 'Osama'
+# app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_DATABASE_DB'] = 'wordsense'
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_DATABASE_USER'] = 'pnkls'
+app.config['MYSQL_DATABASE_PASSWORD'] = 'root'
 app.config['MYSQL_DATABASE_DB'] = 'wordsense'
 
 mysql = MySQL(app)
 conn = mysql.connect()
 cursor = conn.cursor()
 
+english_pos_mapping = {
```

Comment:
I like the use of a dictionary structure but what we had this in a csv file. Then we can add and remove it much easier than having to change the code everytime

Reply:
Yes, I think the same. I think this is what the professor wants as well.

Reply:
Since we decided to add mappings in logic.py file, I added it like a dictionary here. How if we go ahead with this now and if ask others if we want to change.

Reply:
Sounds good

Reply:
sounds good to me too.